techbuzz/src/main/java/com/sivalabs/techbuzz/posts/web/controllers/AddVoteController.java

```diff
@@ -28,4 +30,15 @@ public ResponseEntity<Void> createVote(@Valid @RequestBody CreateVoteRequest req
         postService.addVote(createVoteRequest);
         return ResponseEntity.ok().build();
     }
+
+    @PostMapping("/partials/add-vote")
+    @AnyAuthenticatedUser
+    public String createVote(@Valid @RequestBody CreateVoteRequest request, @CurrentUser User loginUser, Model model) {
+        var createVoteRequest = new CreateVoteRequest(request.postId(), loginUser.getId(), request.value());
+        postService.addVote(createVoteRequest);
+        PostViewDTO post = postService.getPostViewDTO(request.postId());
+        model.addAttribute("post", post);
+        model.addAttribute("abcd", "1234");
```

Comment:
You can remove `model.addAttribute("abcd", "1234");` :-)

Reply:
Thanks for appreciation Siva.